conjure-java-core/src/integrationInput/java/com/palantir/product/SingleUnion.java

```diff
@@ -87,13 +88,14 @@ public String toString() {
 
         private Function<String, T> unknownVisitor;
 
-        public UnknownStageVisitorBuilder<T> foo(Function<String, T> fooVisitor) {
+        public UnknownStageVisitorBuilder<T> foo(@Nonnull Function<String, T> fooVisitor) {
```

Comment:
Do we use Nonnnull anotations elsewhere? I thought we assumed non-null by default unless otherwise specified (and no public conjure components allow nulls).

Reply:
Couldn't find any trace of `@Nonnull` elsewhere in fact, so I dropped it. We still have the `Preconditions.checkNotNull`.